src/redux/auth/auth.slice.ts

```diff
@@ -29,6 +29,8 @@ const authSlice = createSlice({
     logOut: (state) => {
       state.user = null;
       state.token = null;
+
+      localStorage.removeItem('persist:auth');
```

Comment:
use redux-persist, if you will clean redux, it will be clean from cash, no need of manual access to local storage